modules/@apostrophecms/schema/index.js

```diff
@@ -381,6 +381,7 @@ module.exports = {
 
     self.addFieldType({
       name: 'integer',
+      frontComponent: 'string',
```

Comment:
Not sure about the naming of this prop. Open to new ideas..

Reply:
`vueComponent`, and it should be the complete vue component name, it shouldn't go through any more automatic name picking logic. If we are bypassing it then we are bypassing it completely.